docs/api/pool.rst

```diff
@@ -43,7 +43,7 @@ The `!ConnectionPool` class
 .. autoclass:: ConnectionPool
 
    This class implements a connection pool serving `~psycopg.Connection`
-   instances (or subclasses). The constructor has *alot* of arguments, but
+   instances (or subclasses). The constructor has *a lot* of arguments, but
```

Comment:
That's wanted

Reply:
https://www.grammarly.com/blog/a-lot-alot-allot/

Reply:
https://hyperboleandahalf.blogspot.com/2010/04/alot-is-better-than-you-at-everything.html

Reply:
I am aware of the grammar. That was for personal amusement :)